example/mm_jobman.sh

```diff
@@ -208,20 +212,30 @@ submit_each_line_with_mmfloat() {
         if [ "$dryrun" = true ]; then
             full_cmd+="#-------------\n"
         fi
-        cmd="$download_cmd$line$upload_cmd" 
-        full_cmd+="mmfloat submit -i '$image' -j <(echo -e '''$cmd''') -c '$c_value' -m '$m_value' $dataVolume_params\n"
- 
-    done < "$script_file"
 
-    # Remove the last '\n'
-    full_cmd=${full_cmd%\\n}
+        image_run=$(echo $line | awk '{print $3}')
+	subline+="mkdir -p /home/jovyan/TEMP/output && "
+        subline+=$(echo $line | awk '{ print substr($0, index($0,$4)) }')
 
-    # Execute or echo the full command
-    if [ "$dryrun" = true ]; then
-        echo -e "${full_cmd}"  # Replace '&&' with new lines for dry run
-    else
-        eval "$full_cmd"
-    fi
+        #cmd="$download_cmd$subline$upload_cmd"
+	echo "$download_cmd$subline$upload_cmd" > test.sh 
```

Comment:
should echo to standard output then people use it this way: `mm_jobman.sh ... > test.sh` if they need. Dont do it here with hardcoded script names

Reply:
MMC requires a file to execute as a job script, thus the need to put the commands in an external `test.sh` script.